src/strings/PowerFxResources.en-US.resx

```diff
@@ -4694,4 +4694,24 @@
     <value>The specified data source cannot be used with this function.</value>
     <comment>Error Message.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="ErrDivByZero" xml:space="preserve">
+    <value>Invalid operation: division by zero.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrRuntimeTypeMismatch" xml:space="preserve">
+    <value>Runtime type mismatch.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrArgumentOutOfRange" xml:space="preserve">
+    <value>Argument out of range.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidCharValue" xml:space="preserve">
+    <value>Invalid Char value, must be in 1...255 range.</value>
```

Comment:
The invalid value is not a char, it's a number (which will be converted to a char)

Reply:
This is the original error message. This can be changed.